### ISSUE TITLE
add django  as an option for extra_packages

### ISSUE THREAD

**Issue by sloppycoder** [vino9org/cookiecutter-python#3]

**Comment by sloppycoder** [comment]
```text

dependencies = [
   ...,
    "django>=5.0.6",
    "django-ninja>=1.1.0",
    "gunicorn>=22.0.0",
    "dj-database-url>=2.2.0",
    "django-extensions>=3.2.3",
    "psycopg[binary,pool]>=3.1.19",
]

[tool.rye]
dev-dependencies = [
   ...,
    "pytest-django>=4.8.0",
    "django-stubs[compatible-mypy]>=5.0.2",
]

[tool.pytest.ini_options]
...
addopts = "--reuse-db"
...

[tool.mypy]
plugins = ["mypy_django_plugin.main"]

[tool.django-stubs]
django_settings_module = "core_sim.settings"
```

also should change Dockerfile to start gunicorn

```entrypoint.sh
#!/bin/sh

if [ "$DATABASE_URL" = "" ]; then
    echo DATABASE_URL not set, aborting.
    exit 1
fi

python manage.py migrate --check
if [ $? -ne 0 ]; then
    echo running migrations before starting app
    python manage.py migrate
fi

gunicorn -w ${WORKERS:-1} core_sim.wsgi --access-logfile - --bind 0.0.0.0:8000
```

**Comment by sloppycoder** [comment]
implemented with 4c18171
